SimPEG/potential_fields/magnetics/simulation.py

```diff
@@ -355,6 +352,57 @@ def evaluate_integral(self, receiver_location, components):
                     + tmi[1] * node_evals["gyz"]
                     + tmi[2] * node_evals["gzz"]
                 )
+            elif component == "tmi_x":
+                tmi = self.tmi_projection
+                vals_x = (
```

Comment:
For some reason, these lines are highlighted as not being covered by tests, but they are actual ran in `test_forward_Mag_Linear.py`. I don't think this is an issue of this PR, since these lines aren't being marked as covered even in `main`.

I'm mentioning it because it would nice to find out why this is happening and how to solve it.

Reply:
I have no idea why. Maybe @jcapriot would know.

Reply:
I'm marking this conversation as resolved since this coverage issue is not caused by this PR. It should be solved by a different one.